Demos/AIColorPalette/AIColorPalette/AIProxyIntegration.swift

```diff
@@ -9,17 +9,24 @@ import AIProxy // The AIProxy SPM package is found at https://github.com/lzell/A
 import Foundation
 import UIKit
 
-// Visit http://airoxy.pro to get a new partial key and DeviceCheck bypass token.
-//
-// See the README at https://github.com/lzell/AIProxySwift for instructions on
-// adding a DeviceCheck bypass token as an environment variable (which is required
-// for AIProxy to work in the iOS simulator)
-#warning("You must replace the placeholders below")
-let openAIService = AIProxy.openAIService(
-    partialKey: "partial-key-from-your-developer-dashboard",
-    serviceURL: "service-url-from-your-developer-dashboard"
+#warning(
+    """
+    Uncomment one of the methods below. To build and run on device you must follow the AIProxy integration guide.
+    Please see https://www.aiproxy.pro/docs/integration-guide.html")
+    """
 )
 
+/* Uncomment for BYOK use cases */
+ let openAIService = AIProxy.openAIDirectService(
```

Comment:
Do you want to leave this commented like the other examples?

Reply:
Reason I left it uncommented is I get a bunch of errors for ex. "Cannot find 'openAIService' in scope"
![Uploading Screenshot 2024-12-30 at 12.49.35 PM.png…]()

Reply:
New approach of using the `#error` pragma is def better. 🚢